test/parallel/test-v8-serdes.js

```diff
@@ -83,10 +93,14 @@ const deserializerTypeError =
 
   const des = new v8.DefaultDeserializer(ser.releaseBuffer());
   des._readHostObject = common.mustCall(() => {
-    const length = des.readUint32();
-    const buf = des.readRawBytes(length);
-
+    let length = des.readUint32();
+    let buf = des.readRawBytes(length);
     assert.strictEqual(buf.toString(), 'hostObjectTag');
+    for (let i = 0; i < arrayBufferViewsLength; i += 1) {
+      length = des.readUint32();
```

Comment:
Could these be `const length2` and `const buf2` please?
Recycling variables should be avoided.